flowschemas resource is currently skipped by TestApplyResetFields test.

#### What happened:
`flowschemas` resource is currently skipped by `TestApplyResetFields` test.

https://github.com/kubernetes/kubernetes/blob/master/test/integration/apiserver/apply/reset_fields_test.go#L70-L74
```
var resetFieldsSkippedResources = map[string]struct{}{
	// TODO: flowschemas is flaking,
	// possible bug in the flowschemas controller.
	"flowschemas": {},
}
```

When it flakes, it fails with the following error
```
 reset_fields_test.go:288: expected conflict, got error <nil>
```

this is the test result:
```
--- FAIL: TestApplyResetFields (5.94s)
    --- FAIL: TestApplyResetFields/flowcontrol.apiserver.k8s.io/v1beta2,_Resource=flowschemas (0.05s)
    --- PASS: TestApplyResetFields/flowcontrol.apiserver.k8s.io/v1beta2,_Resource=prioritylevelconfigurations (0.01s)
    --- FAIL: TestApplyResetFields/flowcontrol.apiserver.k8s.io/v1beta1,_Resource=flowschemas (0.01s)
    --- PASS: TestApplyResetFields/flowcontrol.apiserver.k8s.io/v1beta1,_Resource=prioritylevelconfigurations (0.01s)
    --- PASS: TestApplyResetFields/flowcontrol.apiserver.k8s.io/v1alpha1,_Resource=flowschemas (0.01s)
    --- PASS: TestApplyResetFields/flowcontrol.apiserver.k8s.io/v1alpha1,_Resource=prioritylevelconfigurations (0.01s)
```


#### What you expected to happen:

We need to identify the root case of why it flakes and fix it. "flowschemas" being skipped means that we may not catch any errors related to field resetting when we add a new version of the API.

#### How to reproduce it (as minimally and precisely as possible):
- remove "flowschemas" from `resetFieldsSkippedResources`
- go test -v -count=1 ./test/integration/apiserver/apply -run="TestApplyResetFields/flowcontrol.apiserver.k8s.io"

run multiple times to see the flake

#### Anything else we need to know?:

#### Environment:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

--- a/test/integration/apiserver/apply/reset_fields_test.go
+++ b/test/integration/apiserver/apply/reset_fields_test.go
@@ -72,11 +72,7 @@ var resetFieldsStatusData = map[schema.GroupVersionResource]string{
 // resetFieldsStatusDefault conflicts with statusDefault
 const resetFieldsStatusDefault = `{"status": {"conditions": [{"type": "MyStatus", "status":"False"}]}}`
 
-var resetFieldsSkippedResources = map[string]struct{}{
-	// TODO: flowschemas is flaking,
-	// possible bug in the flowschemas controller.
-	"flowschemas": {},
-}
+var resetFieldsSkippedResources = map[string]struct{}{}
 
 // noConflicts is the set of reources for which
 // a conflict cannot occur.

